### DISCUSSION TITLE
Schema name -> public

### DISCUSSION THREAD

**Discussion by matejko94** [piccolo-orm/piccolo#568] [Q&A]
I would like to use schema which is not in public and dont know how to change it ?  (postgres schemas)

I have predefind tables I would just like to use piccolo admin pannel on the fastapi. So is it any option to not use piccolo-orm or just way to go with picolo-orm ? Does exist any easier solution ? Also I can use standard connection pool, beacuse we use it ?

**Comment by dantownsend** [comment]
I made a start on a PR: https://github.com/piccolo-orm/piccolo/pull/569. The PR will allow you to do this:

```python
class MyTable(Table, schema="my_schema"):
    ...
```

I need to write some tests etc before releasing it though.

For now, the best solution is to add this to your `PostgresEngine` config:

```python
DB = PostgresEngine({
    'database': 'my_database',
    'server_settings': {
        'search_path': 'my_schema',
    }
})
```

This means Piccolo will work with tables in a schema other than public.

**Reply by dantownsend** [reply]
When working with an existing schema, you need to create Piccolo `Table` classes for your database tables.

You can write these by hand, or use the `piccolo schema generate` command. Here's some more [info in the docs](https://piccolo-orm.readthedocs.io/en/latest/piccolo/tutorials/migrate_existing_project.html#option-2-from-scratch).
